tests/comparisons/Plugin/testMainUpdateComposer.json

```diff
@@ -1,12 +1,12 @@
 {
     "autoload": {
         "psr-4": {
-            "ComposerExample\\": "./plugins/ComposerExample/src"
+            "ComposerExample\\": "/tmp/tests/BakedPlugins/ComposerExample/src"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "ComposerExample\\Test\\": "./plugins/ComposerExample/tests"
+            "ComposerExample\\Test\\": "/tmp/tests/BakedPlugins/ComposerExample/tests"
```

Comment:
Paths should be relative as when you deploy the path will probably not be the same.